Kiss.Bff.EndToEndTest/ContactMomentSearch/ContactMomentSearchScenarios.cs

```diff
@@ -111,11 +111,70 @@ public async Task SearchByBSN_Invalid()
 
             await Page.PersonenThird_SearchButton().ClickAsync();
 
-            await Step("Then the message is displayed as “Dit is geen valide BSN.”");
-             
-            Assert.AreEqual( await Page.PersonenBsnInput().EvaluateAsync<string>("(el) => el.validationMessage"), "Dit is geen valide BSN.");
+            await Step("Then the message is displayed as “Dit is geen valide BSN.”"); 
+
+            Assert.AreEqual(await Page.PersonenBsnInput().EvaluateAsync<string>("(el) => el.validationMessage"), "Dit is geen valide BSN.");
+
+        }
+
+
+        [TestMethod("5. Searching by Postcode and Huisnummer (Valid)")]
+        public async Task SearchByPostcodeAndHuisnummer_Valid()
+        {
+            await Step("Given the user is on the startpagina ");
+
+            await Page.GotoAsync("/");
+
+            await Step("When user starts a new contactmoment");
+
+            await Page.CreateNewContactmomentAsync();
+
+            await Step("And user enters \"3544NG\" in the field Postcode and “10” in field Huisnummer");
+
+            var postCode = "3544 NG";
+            var huisNummer = "10";
+
+            await Page.Personen_PostCodeInput().FillAsync(postCode);
+            await Page.Personen_HuisnummerInput().FillAsync(huisNummer);
+
+            await Step("And clicks the search button");
+
+            await Page.PersonenSecond_SearchButton().ClickAsync();
+
+
+            await Step("Then a list of multiple records associated with same huisnummer and postcode is displayed ");
+
+            await Page.GetByRole(AriaRole.Table).WaitForAsync();
```

Comment:
Can we use Expect here? In case of a failing test, it gives a bit nicer feedback

Reply:
if we use expect we only check specific count, but in this case its unknown multiple values

Reply:
Yeah it's too bad playwright doesn't have Expect(...).ToHaveCountGreaterThan(). But, I think we can do this in stead of the WaitForAsync, right?
```suggestion
            await Expect(Page.GetByRole(AriaRole.Table)).ToBeVisibleAsync();
```

Reply:
sure, i have updated based on the comment